client/webserver/site/src/js/markets.ts

```diff
@@ -605,8 +605,11 @@ export default class MarketsPage extends BasePage {
    * supported
    */
   assetsAreSupported () {
-    const [b, q] = [this.market.base, this.market.quote]
-    return b && q
+    const [base, quote] = [this.market.base, this.market.quote]
+    if (!base || !quote) return false
+    const { baseCfg: b, quoteCfg: q } = this.market
```

Comment:
I would just leave this as `baseCfg` and `quoteCfg` to be able to more clearly differentiate in the check below.